examples/spouts/spout101/README.md

```diff
@@ -0,0 +1,181 @@
+# Spout Pipelines 
+>![pach_logo](./img/pach_logo.svg) This new implementation of the spout functionality is available in version **1.12 and higher**.
+
+
+## Intro
+A spout is a type of pipeline that ingests 
+streaming data (message queue, database transactions logs,
+event notifications... ). 
+See spout pipelines as a bridge
+between an external stream of data and Pachyderm's repo.
+In Martin Fowler's enterprise integration patterns' terms,
+a Pachyderm spout is a 
+*[Polling Consumer](https://www.enterpriseintegrationpatterns.com/patterns/messaging/PollingConsumer.html)* 
+(actively polls for a message, processes it, then polls for another).
+
+Generally, you use spouts for situations 
+when the interval between new data generation
+is large or sporadic, but the latency requirement
+to start the processing is short. 
+Therefore, a regular pipeline with a cron input
+that polls for new data might not be an optimal solution.
+
+A spout pipeline has three major differences from regular pipelines:
+- it runs continuously
+- it does not take an input repo. Instead, it consumes messages from a third-party messaging system.
+- `pfs/out` is not directly accessible. You will need to use `pachctl put file` to write into your output repo. In this example, we have used the `pachyderm-python` library. 
+
+
+You configure a spout in the 
+[pipeline specification](https://docs.pachyderm.com/latest/reference/pipeline_spec/)
+file by adding a `spout` section after its pipeline section.
+At a minimum, a spout pipeline will need a pipeline name,
+an empty spout section, and a transform section
+where you will specify the transformation you want
+to execute on the messages consumed.
+
+>![pach_logo](./img/pach_logo.svg) Note that the use
```

Comment:
do we have some methodology to track notices like this such that when build + spouts work we remember to remove this?

Reply:
I have created a feature request for this: **Build pipelines support for spouts** #5560, the merge of which will alert me.